products/coldfusion.md

```diff
@@ -18,32 +18,49 @@ releases:
     releaseDate: 2023-05-17
     eol: 2028-05-16
     extendedSupport: 2029-05-16
-    link: null # does not exist yet, will probably become valid after the first update
+    latestReleaseDate: 2023-10-05
+    latest: "ColdFusion (2023 Release) Update 5"
```

Comment:
The `Build Number` would be more appropriate here. For example 2023.0.5.330608 for this version (see https://helpx.adobe.com/fr/coldfusion/kb/coldfusion-2023-updates.html).

Note that I replaced the `,` by `.` in build numbers. The reason behind that is to make it possible to automate latest release update in the future.

Reply:
Merci! Great idea for future automation. I just made another commit with the build numbers added separated by periods.